client/context/query.go

```diff
@@ -325,41 +194,43 @@ func (ctx CLIContext) query(path string, key cmn.HexBytes) (res []byte, err erro
 	return resp.Value, nil
 }
 
-// Certify verifies the consensus proof at given height
+// Certify verifies the consensus proof at given height.
 func (ctx CLIContext) Certify(height int64) (lite.Commit, error) {
 	check, err := tmliteProxy.GetCertifiedCommit(height, ctx.Client, ctx.Certifier)
-	if tmliteErr.IsCommitNotFoundErr(err) {
+	switch {
+	case tmliteErr.IsCommitNotFoundErr(err):
 		return lite.Commit{}, ErrVerifyCommit(height)
-	} else if err != nil {
+	case err != nil:
 		return lite.Commit{}, err
 	}
+
 	return check, nil
 }
 
-// verifyProof perform response proof verification
-// nolint: unparam
-func (ctx CLIContext) verifyProof(path string, resp abci.ResponseQuery) error {
-
+// verifyProof perform response proof verification.
+func (ctx CLIContext) verifyProof(_ string, resp abci.ResponseQuery) error {
 	if ctx.Certifier == nil {
-		return fmt.Errorf("missing valid certifier to verify data from untrusted node")
+		return fmt.Errorf("missing valid certifier to verify data from distrusted node")
```

Comment:
Is there a difference in meaning intended here or did we just change the word? AFAIK "untrusted" is the more common term.

Reply:
Not really apart from common nomenclature.

Reply:
OK nbd